moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/parameterization/joint_space/constrained_planning_state_space.h

```diff
@@ -48,11 +48,11 @@ MOVEIT_CLASS_FORWARD(ConstrainedPlanningStateSpace);
  * The state type of a constrained state space in OMPL is a wrapper around a normal
  * state type. Therefore we need to unwrap it before we can copy the joint values
  * between OMPL and MoveIt.
- * 
+ *
```

Comment:
Did you run clang-format?

Reply:
Yup, but not clang-tidy. Looks like I was missing some override statements.